shootout/shootout.py

```diff
@@ -3,28 +3,27 @@
 # https://github.com/Redjumpman/Jumper-Plugins/blob/a5a55e3968cb366bf257cb0e886a1c30588e85ef/russianroulette/russianroulette.py
 #
 from redbot.core import bank, commands, checks, Config
-import discord
-import asyncio
+import asyncio, contextlib, discord, random, shlex
 
 
 class Shootout(commands.Cog):
     default_config = {
         "cost": 50,
-        "message": "Shoot!",
-        "wait_time": 60,
+        "Messages": ["Shoot!", "Draw!", "Fire!", "Kill!"],
         "Session": {"Pot": 0, "Players": [], "Active": False},
```

Comment:
So your code to play games works in a single channel, but the way Session is setup means only one game at a time can be played per guild. You should set session to the channel config, so every text channel can have an independent game going. Also need to then change the admin clear_game function to clear game for the channel

Reply:
Implemented multi-channel support.